secp256k1-sys/src/lib.rs

```diff
@@ -365,7 +386,7 @@ unsafe fn strlen(mut str_ptr: *const c_char) -> usize {
 /// Rust doesn't promise what pointers does it give to ZST (https://doc.rust-lang.org/nomicon/exotic-sizes.html#zero-sized-types-zsts)
 /// In case the type is empty this trait will give a NULL pointer, which should be handled in C.
 /// 
-pub(crate) trait CPtr {
+pub trait CPtr {
```

Comment:
In a late PR I'll try move this to `secp256k1` and not `-sys` and make it `pub(crate)` again.

Reply:
Is this not used in -sys? In that case I can move it.. The idea is to try and reduce the times we need to bump a -sys version. So I think better make breaking changes now.

Reply:
Hmm, seems to be used in macros as well. Yeah I think I'd prefer to merge this as is :)

Reply:
Merge != release.
let's not change this PR any more so it can get merged at some point :)